recipe/meta.yaml

```diff
@@ -9,11 +9,10 @@ source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
   sha256: 4235ffeb605bcd4e22e716577f7d61e4aa1bc82f65276bb542701a81b7933356
   patches:
-    - patch
+    - patch  # [linux]
```

Comment:
Note it's fine to only patch Linux since the patch modifies the `else:` condition in `setup.py` that only affects Linux:
```py
if platform == 'darwin':
    if not 'ARCHFLAGS' in os.environ:
        os.environ['ARCHFLAGS'] = '-arch x86_64'
    extra_compile_args.append('-w')
    if use_ice:
        libraries = ["IceSSL", "Ice", "Slice", "IceUtil"]
        extra_link_args = []
    else:
        libraries=['iconv']
        extra_link_args = ['-framework','Security', '-framework','CoreFoundation']

    def filterName(path):
        d = os.path.dirname(path)
        if use_ice and d.find("src/ice/") != -1:
            return False
        if d.find('bzip2') != -1:
            return False # Don't compile the bzip2 source under darwin or linux.
        return True

elif platform == 'win32':
    extra_link_args = []
    libraries=[]
    define_macros.append(('WIN32_LEAN_AND_MEAN', None))
    define_macros.append(('ICE_BUILDING_ICE_UTIL', None))
    define_macros.append(('ICE_BUILDING_SLICE', None))
    define_macros.append(('ICE_BUILDING_ICE', None))
    define_macros.append(('ICE_BUILDING_ICE_SSL', None))
    define_macros.append(('_WIN32_WINNT', '0x601'))
    include_dirs.append('src/ice/bzip2')
    extra_compile_args.append('/EHsc')
    extra_compile_args.append('/wd4018')
    extra_compile_args.append('/wd4146')
    extra_compile_args.append('/wd4244')
    extra_compile_args.append('/wd4250')
    extra_compile_args.append('/wd4251')
    extra_compile_args.append('/wd4267')
    extra_compile_args.append('/wd4275')
    extra_compile_args.append('/wd4996')
    libraries=['dbghelp', 'Shlwapi', 'rpcrt4','advapi32','Iphlpapi','secur32','crypt32','ws2_32']
    # SysLoggerI.cpp shouldn't be built under Windows.
    def filterName(path):
        b = os.path.basename(path)
        if b == 'SysLoggerI.cpp':
            return False
        return True

else:
    #
    # TODO: Get rid of this hack to remove -Wstrict-prototypes from the compiler options
    # when http://bugs.python.org/issue1222585 is fixed. Note that this hack doesn't work
    # with recent distutils versions which no longer allow overriding OPT in the env.
    #
    from distutils.sysconfig import get_config_vars
    (opt,) = get_config_vars('OPT')
    os.environ['OPT'] = " ".join(flag for flag in opt.split() if flag != '-Wstrict-prototypes')

    extra_compile_args.append('-w')
    extra_link_args = []
    if use_ice:
        libraries = ["IceSSL", "Ice", "Slice", "IceUtil"]
    else:
        libraries=['ssl', 'crypto', 'bz2', 'rt']
        if platform is not 'freebsd':
            libraries.append('dl')

    def filterName(path):
        d = os.path.dirname(path)
        if use_ice and d.find("src/ice/") != -1:
            return False
        if d.find('bzip2') != -1:
            return False # Don't compile the bzip2 source under darwin or linux.
        return True
```